features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/Commons/AddRoleWizard.jsx

```diff
@@ -171,8 +171,8 @@ export default function AddRoleWizard(props) {
         <AddItem
             onSave={onAddRole}
             onClose={onClose}
-            title='Add new role permissions'
-            buttonText='Add role permissions'
+            title='Add new scope mapping'
+            buttonText='Add scope mapping'
```

Comment:
Missing i18n

Reply:
Please find and fix in the other places accordingly as well

Reply:
Fixed with 46a151e